src/test/resources/features/comp/COMPTriage.feature

```diff
@@ -60,10 +107,30 @@ Feature: Service Triage
     And a case closure note should be visible showing the reason for closure
     And the read-only Case Details accordion should contain all case information entered during the "Service Triage" stage
 
+  @COMPWorkflow @COMPRegression
```

Comment:
Is there a reason to comment out this scenario or shall we delete the scenario

Reply:
I designed the scenario according to how the other complaint type functionality works (this complaint type is different for some reason), I've asked Casey if this is correct so based on his response I'll either leave it commented out or delete it.